src/status_im/utils/clocks.cljs

```diff
@@ -94,6 +94,5 @@
   (inc (max local-clock (->timestamp-bid))))
 
 (defn receive [message-clock local-clock]
-  (-> (+ 1000 (max (or message-clock 0) (or local-clock 0)))
-      safe-timestamp
-      inc))
+  (-> (max (or message-clock 0) (or local-clock 0))
+      safe-timestamp))
```

Comment:
I noticed that we don't increment it here anymore, is that expected?

Reply:
Yes, it's not needed really, it's in the standard algorithm https://en.wikipedia.org/wiki/Lamport_timestamps so that's why it was there in the first place, but we are not interested in "receive" events, so it can be skipped (as long as we increment it on send we are good).

Reply:
then sounds good!